Geocortex.Mobile.Samples/Geocortex.Mobile.Samples/Samples/AppConfiguration/MapAndFeatureCommands/map-and-feature-commands.xml

```diff
@@ -114,6 +164,7 @@
     <gxm:taskbar grow="3">
 
       <map config="map1" slot="main">
+		<gxm:geometry-toolbar slot="top-center-visible" id="geometry-toolbar" config="geometry" margin="0.8" style="round" active="false" />
```

Comment:
Indentation is off

Reply:
Tabs vs. spaces. A tale as old as time. Should be fixed now.